File: ABB/src/abb/NodoArbol.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package abb;

/**
 *
 * @author Usuario
 */
public class NodoArbol {
    private NodoArbol derecho;
    private NodoArbol izquierdo;
    private int dato;

    public NodoArbol(int dato) {
        this.derecho = null;
        this.izquierdo = null;
        this.dato = dato;
    }

    public NodoArbol getDerecho() {
        return derecho;
    }

    public void setDerecho(NodoArbol derecho) {
        this.derecho = derecho;
    }

    public NodoArbol getIzquierdo() {
        return izquierdo;
    }

    public void setIzquierdo(NodoArbol izquierdo) {
        this.izquierdo = izquierdo;
    }

    public int getDato() {
        return dato;
    }

    public void setDato(int dato) {
        this.dato = dato;
    }
    
    
}

File: Taller2_2018/NodoArbol.cpp
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */

/* 
 * File:   NodoArbol.cpp
 * Author: Juan Pablo Martinez
 * 
 * Created on 5 de junio de 2018, 20:01
 */

#include "NodoArbol.h"
#include <cstdlib>
NodoArbol::NodoArbol(int dato) {
    this->dato=dato;
    this->der =NULL;
    this->izq =NULL;
}

int NodoArbol::getDato() {
    return dato;
}

NodoArbol* NodoArbol::getDerecho() {
    return der;
}

NodoArbol* NodoArbol::getIzquierdo() {
    return izq;
}

void NodoArbol::setDato(int dato) {
    this->dato=dato;
}

void NodoArbol::setDerecho(NodoArbol *der) {
    this->der=der;
}

void NodoArbol::setIzquierdo(NodoArbol *izq) {
    this->izq=izq;
}



NodoArbol::~NodoArbol() {
}

